src/NCalc.Core/Helpers/MathHelperOptions.cs

```diff
@@ -30,6 +30,12 @@ public bool AllowCharValues
         get => options.HasFlag(ExpressionOptions.AllowCharValues);
     }
 
+    public bool LongAsDefault
+    {
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        get => options.HasFlag(ExpressionOptions.LongAsDefault);
```

Comment:
if I remember correctly the `HasFlag` can be notoriously inefficient depending on the runtime... if you target netstandard you might want to consider just bit operations

Reply:
Hi, nice to see your here!

This doesn't seems be an issue, because we're running benchmark on .net 8.0, and the first PR with adding this new option doesn't have any perf impact: https://github.com/ncalc/ncalc/actions/runs/18692621173/job/53301871481

Seems to be a`Select` parser issue